client/my-sites/post-type-list/index.jsx

```diff
@@ -253,6 +255,7 @@ class PostTypeList extends Component {
 		} );
 		const showUpgradeNudge =
 			siteId &&
+			! this.props.isVipSite &&
```

Comment:
Nitpick: I'd move this check to a `isVipSite` variable along with `query`, `siteId`, ..., in line 249 just to be aligned with the rest of the props.

Reply:
Thanks for catching that.